debezium-core/src/main/java/io/debezium/connector/common/CdcSourceTaskContext.java

```diff
@@ -20,13 +24,17 @@
  */
 public class CdcSourceTaskContext {
 
+    private static final String[] EMPTY_CAPTURED_LIST = new String[0];
+
     private final String connectorType;
     private final String connectorName;
     private final Clock clock;
+    private final Supplier<Collection<? extends DataCollectionId>> collectionsSupplier;
 
-    public CdcSourceTaskContext(String connectorType, String connectorName) {
+    public CdcSourceTaskContext(String connectorType, String connectorName, Supplier<Collection<? extends DataCollectionId>> collectionsSupplier) {
```

Comment:
Because it is a dynamic value
1) This information might not be available in snapshotting phase for binlog
2) The new tables can be created over the time that matches whitelist - a typical example for SQL Server is enabling capture process for a new table

Reply:
> Because it is a dynamic value

Ah yes, that makes sense.